pkg/cmd/pipeline/create.go

```diff
@@ -43,6 +44,31 @@ func NewCmdPipelineCreate(f *factory.Factory) *cobra.Command {
 				return err
 			}
 
+			clusterMap := getClusters(cmd.Context(), f)
+			clusterNames := make([]string, 0, len(clusterMap))
+
+			for name := range clusterMap {
+				clusterNames = append(clusterNames, name)
+			}
+			if len(clusterNames) > 0 {
+				prompt := &survey.Select{
```

Comment:
`survey` is part of charmbracelet/bubbletea which we plan to stop using as part of the CLI modernization initiative, so we might want to find an alternative. t